spec/counter_culture_spec.rb

```diff
@@ -1371,7 +1371,8 @@
     categ.reload.posts_count.should == 1
   end
 
-  describe "#previous_model" do
+  # TODO check that effect is tested rather than method directly
```

Comment:
I haven't gone back to check that the effect of needing the previous model in the dynamic case is handled in the specs. If so I will just remove that spec.